_posts/2019-09-16-how-to-run-rspec-on-github-actions-for-ruby-on-rails-app-using-parallel-jobs.md

```diff
@@ -114,8 +114,10 @@ I also recorded video showing how it all works and how CI builds with parallel j
 
 If you would like to better understand how Queue Mode works in Knapsack Pro and what else problems it solves you will find a few useful information in below video.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/hUEB1XDKEFY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/hUEB1XKDEFY" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
you accidentally change url to invalid yt video. Here is correct one https://www.youtube.com/embed/hUEB1XDKEFY

Reply:
:O Wow, great catch. Thanks